projectroles/forms.py

```diff
@@ -444,6 +446,18 @@ def _set_app_setting_notes(self, s_field, s_val):
                 self.fields[
                     s_field
                 ].help_text += ' [Not editable on target sites]'
+        if plugin:
```

Comment:
Since there's quite a few rows of repetition here, how about creating a `get_app_setting_label()` helper in `SODARModelForm` and calling that from both the project and user forms?